fretboard/src/main/java/mozilla/components/service/fretboard/Fretboard.kt

```diff
@@ -43,4 +47,17 @@ class Fretboard(
             // Keep using the local experiments
         }
     }
+
+    /**
+     * Checks if the user is part of
+     * the specified experiment
+     *
+     * @param context context
+     * @param descriptor descriptor of the experiment to check
+     *
+     * @return true if the user is part of the specified experiment, false otherwise
+     */
+    fun isInExperiment(context: Context, descriptor: ExperimentDescriptor): Boolean {
+        return evaluator.evaluate(context, descriptor, experiments)
+    }
```

Comment:
This is a good simple API we can start with (and land).

In a follow-up it would be nice to explore other more Kotlin-y and useful variations. In an app I often want to do something if I am part of an experiment. So we could make this easier for callers and create something like:

```Kotlin
fun withExperiment(context: Context, descriptor: ExperimentDescriptor, block: (Experiment) -> Unit)
```

And I could use that in an app like (Experiments holding ExperimentDescriptor constants in this case):

```Kotlin
Fretboard.withExperiment(context, Experiments.ONBOARDING_A) { experiment -> 
   // ...
}
```

Or as an extension method on the descriptor:

```Kotlin
Experiments.SHOW_DIALOG.isActive { experiment ->
  // ...
}
```

^-- All names above are just made up and there may be better options to name things :)